lisa/operating_system.py

```diff
@@ -1220,6 +1220,25 @@ def install_epel(self) -> None:
             sed = self._node.tools[Sed]
             sed.substitute("$releasever", "8", "/etc/yum.repos.d/epel*.repo", sudo=True)
 
+    def _verify_package_result(self, result: ExecutableResult, packages: Any) -> None:
+        # yum returns exit_code=1 if DNF handled an error with installation.
+        # We do not want to fail if exit_code=1, but warn since something may
+        # potentially have gone wrong.
+        if result.exit_code == 1:
+            self._log.debug(f"DNF handled error with installation of {packages}")
+        elif result.exit_code == 0:
+            self._log.debug(f"{packages} is/are installed successfully.")
+        else:
+            raise LisaException(
+                f"Failed to install {packages}. exit_code: {result.exit_code}"
```

Comment:
I think we should also log stderr and stout

Reply:
This is the code as it exists currently, I'm just hoisting it to the superclass here. Can we make that a separate PR?

Reply:
I think too we usually don't log the full stdout and stderr for these because debug level will already log that, unless we've explicitly disabled logging there

Reply:
If there are a lot of log, it's not good to output in exception. But if it's possible, the key information should be extracted, and added in the exception. But it should be done in another PR, if it's in priority.

Reply:
Sure! This can be done in another PR.